lib/ui/drawing_space/bottom_brush_tool_options.dart

```diff
@@ -0,0 +1,88 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+import 'package:paintroid/tool/src/brush_paint.dart';
+import 'package:paintroid/ui/shared/custom_action_chip.dart';
+
+class BottomBrushToolOptions extends ConsumerStatefulWidget {
+  const BottomBrushToolOptions({super.key});
+
+  @override
+  ConsumerState<BottomBrushToolOptions> createState() =>
+      _BottomBrushToolOptionsState();
+}
+
+class _BottomBrushToolOptionsState
+    extends ConsumerState<BottomBrushToolOptions> {
+  late Color _roundChipBackgroundColor;
+  late Color _squareChipBackgroundColor;
```

Comment:
Should we give them a default value so they do not have to be late?
Like round defaults to blue and square to white?

Reply:
Done :+1: